packages/tools/client-debugger/client-debugger-view/src/components/AudienceView.tsx

```diff
@@ -89,158 +88,81 @@ export function AudienceView(props: AudienceViewProps): React.ReactElement {
 		return <Waiting label="Waiting for Audience data." />;
 	}
 
+	// TODO: Determine if myClientMetaData is necessary
 	const myClientMetadata = audienceData.audienceState.find(
 		(audience) => audience.clientId === audienceData.clientId,
 	)?.client;
 
+	const audienceStateItems = audienceStateDataFilter(
+		audienceData.audienceState,
+		myClientMetadata,
+	);
+	const audienceHistoryItems = audienceHistoryDataFilter(audienceData.audienceHistory).reverse();
+
 	return (
-		<Stack
-			styles={{
-				root: {
-					height: "100%",
-				},
-			}}
-		>
-			<StackItem>
-				<h3>Audience members: ({audienceData.audienceState.length})</h3>
-				<MembersView
-					audience={audienceData.audienceState}
-					myClientId={audienceData.clientId}
-					myClientConnection={myClientMetadata}
-				/>
-			</StackItem>
-			<StackItem>
-				<h3>History</h3>
-				<HistoryView history={audienceData.audienceHistory} />
-			</StackItem>
-		</Stack>
+		<>
+			<Divider appearance="brand"> Audience State </Divider>
+			<AudienceStateTable audienceStateItems={audienceStateItems} />
+			<Divider appearance="brand"> Audience History </Divider>
+			<AudienceHistoryTable audienceHistoryItems={audienceHistoryItems} />
+		</>
 	);
 }
 
 /**
- * {@link MembersView} input props.
+ * Filtered audience state data for {@link audienceStateDataFilter}
  */
-interface MembersViewProps {
-	/**
-	 * The current audience
-	 */
-	audience: AudienceClientMetadata[];
-
-	/**
-	 * My client ID, if the Container is connected.
-	 */
-	myClientId: string | undefined;
-
-	/**
-	 * My client connection data, if the Container is connected.
-	 */
+export interface FilteredAudienceStateData {
+	clientId: string;
+	userId: string;
+	mode: string;
+	scopes: string[];
 	myClientConnection: IClient | undefined;
 }
 
 /**
- * Displays a list of current audience members and their metadata.
+ * Removes unncessary data in audienceData.audienceState
  */
-function MembersView(props: MembersViewProps): React.ReactElement {
-	const { audience, myClientId, myClientConnection } = props;
-
-	const transformedAudience = combineMembersWithMultipleConnections(audience);
-
-	const memberViews: React.ReactElement[] = [];
-	for (const member of transformedAudience.values()) {
-		memberViews.push(
-			<StackItem key={member.userId}>
-				<AudienceMemberView
-					audienceMember={member}
-					myClientId={myClientId}
-					myClientConnection={myClientConnection}
-				/>
-			</StackItem>,
-		);
-	}
-
-	return <Stack>{memberViews}</Stack>;
+function audienceStateDataFilter(
```

Comment:
I generally recommend naming functions to start with a verb, so it's clear that it represents an action of some kind. E.g. "filterAudienceStateData".

That said, since this is only used in 1 place, we could probably just inline the call to `.map` into the component too.

Reply:
Based on the earlier conversation with @alexvy86, I changed the name to `transformAudienceStateData`.